index.rst

```diff
@@ -3,7 +3,7 @@
 nRF Asset Tracker
 #################
 
-The nRF Asset Tracker aims to provide a concrete end-to-end example for an ultra-low power cellular IoT product in the asset tracker space, specifically a Cat Tracker.
+The nRF Asset Tracker is a web application that aims to provide a concrete end-to-end example for an ultra-low power cellular IoT product in the asset tracker space, specifically a Cat Tracker.
```

Comment:
In the entire documentation, we don't really define what nRF Asset Tracker is. Yes, we are saying what it aims to provide and what it enables, but not really what it is. @coderbyheart feel free to improve this, as I'm not totally sure at all.

Reply:
Seems that the build is failing, probably because the line 101 in the ..azure/ContinuousIntegration.rst. Maybe .. code:: json needs to be ..code-block::

Reply:
Fixed in saga.